Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Container image that runs our code
+FROM node:10
```

Comment:
Do you know if can we include Powershell core with the docker image on setup?

would be nice pre-setup if the users would like to run powershell scripts.

Reply:
I am looking at this https://github.com/PowerShell/PowerShell-Docker/blob/master/release/preview/arm32v7/docker/Dockerfile. Should that help to bring PowerShell 6 to the action?

Reply:
Sorry @VelinGeorgiev my knowledge on Docker is very limited. Maybe @waldekmastykarz can help here?

Reply:
You can only have one base image so I believe that the approach to take would be to change the base image to use something like Ubuntu and then install Node, PowerShell Core and then the CLI.